README.md

```diff
@@ -1 +1,131 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Star Educational Society] <a name="about-project">
+Hello , this a website for Star Educational Center</a>
+
+[Star's Website] This a webasite mobile first app. It is created according to capston project but the contents are different. For now just about and home page are completed but the complete version will be pushed next time.
+
+## 🛠 Built With 
+
+### Tech Stack 
+HTML
+CSS
+JAVASCRIPT
+BOOTSTRAP
+
+### Key Features 
+
+- **[Navbar]**
+- **[Background image]**
+- **[Mobile first]**
+- **[Linter is used]**
+- **[Eslinter is used]**
+- **[Responsive]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:You have to clone this project to your local machine
+
+### Setup
+
+Clone this repository to your desired folder:git clone https://github.com/Elhambasir/capstone-project1--conference-page.git
+
+### Install
+
+Install this project with: Any software which you are comfortable with
+
+### Usage
+
+To run the project, execute the following command: On your computer's keyboard press f5 or run it with live server
+
+### Run tests
+
+To run tests, run the following command: In visual studio code with live server plugin right click and run live server.
+
+### Deployment
+
+## 🚀 Live Demo <a name="Live Demo"></a>
+
+[Live Demo Link](https://elhambasir.github.io/capstone-project1--conference-page/)
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@Elhambasir](https://github.com/Elhambasir)
+- LinkedIn: [basir ahmadi](https://www.linkedin.com/in/basir-ahmadi-8aba89263)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+## Future Features
+
+- [ ] **[Fully responsive]**
+- [ ] **[Other features]**
+- [ ] **[Better style]**
+- [ ] **[Complete other parts]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+## Contributions
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+If you like this project reach me on my contact: WhatsApp => +93767405466
+
+
```

Comment:
- [optional] you are missing the `show your support` and `License` headers in your README. Please kindly edit to make them visible to make your file clean and professional

Reply:
Thank you